server/internal/usecase/interactor/item.go

```
@@ -587,50 +596,54 @@
-		items, err := i.repos.Item.FindByIDs(ctx, lo.Keys(itemsEvent), nil)
-		if err != nil {
-			return interfaces.ImportItemsResponse{}, err
-		}
-		var events []Event
+		return res.Into(), nil
+	}
 
-		for k, changes := range itemsEvent {
-			vi := items.Item(k)
-			it := vi.Value()
+	runRes, err := Run1(ctx, operator, i.repos, Usecase().Transaction(), f)
```

golangci-lint (1.62.2)

ineffectual assignment to err

(ineffassign)